lte/gateway/c/oai/common/log.h

```diff
@@ -151,6 +154,7 @@ typedef enum {
   LOG_NGAP,
   LOG_MME_APP,
   LOG_NAS,
+  LOG_NAS5G,
```

Comment:
should this be LOG_NAS_5G? fits better I think with the naming

Reply:
I  will pick this up in the later PRs to be cherry-picked.